sunpy/map/map_factory.py

```diff
@@ -190,7 +203,7 @@ def _validate_meta(self, meta):
         else:
             return False
 
-    def _parse_args(self, *args, silence_errors=False, **kwargs):
+    def _parse_args(self, *args, filetype=None, silence_errors=False, **kwargs):
```

Comment:
Does this need to be passed in? Would it not be covered by "**kwargs"?

Reply:
Yes, I thought explicit keyword arg will be better for readability and further developments, will do the change👍

Reply:
That is true, but if I am honest, I dislike the entire concept of the "filetype" keyword in our IO module.

Reply:
I can understand, I'll do the needful changes, 
and should I close this and open a new PR, to avoid a commit mess?